gatsbyUtils/createPages.js

```diff
@@ -1014,26 +1014,30 @@ const generateDocHome = (
     };
   });
 
-  const newestBlog = {
-    cn: list
+  const getTwoNewestBlog = lang => {
+    return list
       .sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime())
-      .filter(i => i.fileLang === 'cn')[0],
-    en: list
-      .sort((a, b) => new Date(b.date).getTime() - new Date(a.date).getTime())
-      .filter(i => i.fileLang === 'en')[0],
+      .filter(i => i.fileLang === lang)
+      .slice(0, 2);
   };
 
   homeData.forEach(({ language, data, path, version }) => {
     const isBlog = checkIsblog(path);
     const editPath = path.split(language === 'en' ? '/en/' : '/zh-CN/')[1];
-    const link = versionInfo[version]['pymilvus'];
-    // generate pyMilvus API link according to different version
-    // and render it at Doc home Recommended Articles section last item of Get started
-    const pyMilvus = {
-      text: 'PyMilvus API Reference',
-      link: `/api-reference/pymilvus/${link}/install.html`,
-    };
-    data.section3.items[0].list.push(pyMilvus);
+    data.section3.items.forEach(item => {
+      item.list.forEach(subItem => {
+        const { link } = subItem;
+        if (link.includes('{{') && link.includes('}}')) {
+          const head = link.match(/(\S*)\{{/)[1];
+          // must be one of pymilvus,go node java
+          const language = link.match(/\{{(\S*)\}}/)[1].split('_')[0];
```

Comment:
Need more comment, like what we need to write in home.json. 
And if they write wrong language, we can throw error to stop building website here.